### ISSUE TITLE
Have way to debug both native iOS and Dart code in same session

### ISSUE THREAD

**Issue by tdevaux** [flutter/devtools#4507]
Developing plugins, it'd be nice to have a way to debug both Objc/Swift code AND Dart code.

Ex:
- Be able to put breakpoints in XCode or Android Studio, and break on either.
- Or; in DevTools, have the option to debug native code and Dart.

**Comment by kenzieschmoll** [comment]
This is possible (g3 [link](devtools-g3-link-1)). We should think about if we can make this an easier workflow for users or at least make sure this is documented on flutter.dev

**Comment by srawlins** [comment]
@kenzieschmoll that g3 link is dead. Do you think this is still possible? To me this seems out-of-scope for the project.

**Comment by kenzieschmoll** [comment]
Yeah let's close as not planned. This is would need to be supported at the vm debugger level somehow.
